File: backend/art-galery/src/main/java/com/unmsm/movil/tecnisis/art_galery/infrastructure/adapters/input/rest/model/response/PersonResponse.java
package com.unmsm.movil.tecnisis.art_galery.infrastructure.adapters.input.rest.model.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Builder @Getter
@NoArgsConstructor @AllArgsConstructor
public class PersonResponse {
    private Long id;
    private String name;
    private String dni;
    private String address;
    private String gender;
    private String phone;
    private String role;
}

File: tecnisis-app/app/src/main/java/com/example/tecnisis/data/person/PersonResponse.kt
package com.example.tecnisis.data.person

import com.example.tecnisis.data.user.UserResponse

data class PersonResponse(
    val id: Long,
    val name: String,
    val dni: String,
    val address: String,
    val gender: Char,
    val phone: String,
    val role: String,
    val user: UserResponse
)
